### ISSUE TITLE
Function openImagesAsModals isn't properly quoted

### ISSUE THREAD

**Issue by double-beep** [soscripted/sox#417]
I recently saw the new feature Open as modals, but I realized that some reversed keywords aren't quoted at all, like `css`, `id`, etc. This can lead to "Unexpected identifier" errors.

To make it compatible with older versions of browsers, is it possible to fix that?

**Comment by shu8** [comment]
@double-beep Thanks for the bug report!

Do you mean the keys for the object properties themselves should be quoted? I wasn't aware this was incompatible with some browsers if so! Please could you post your browser/platform so I can try an reproduce it and confirm if the fix works? :)

**Comment by double-beep** [comment]
Yes, the keys should be quoted. Some are reversed keywords in IE-like browsers and are likely to cause this error:

```
Syntax error @ "Stack Overflow Extras (SOX)"!
##########################
JSHINT output:
##########################

SyntaxError: Expected identifier, string or number
   at C.E_c (Unknown script code:3:391)
   at ja (Function code:62:80)
   at create (Function code:73:419)
   at e (Function code:14:351)
```
While I am using Chrome, this is only at home. At work, I have to use ME, v44.17763.1.0

**Comment by shu8** [comment]
Thanks for clarifying!

Turns out Edge [doesn't support the spread operator in object literals yet](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Browser_compatibility) :(

Fixed in [dev 2.5.19](https://github.com/soscripted/sox/raw/dev/sox.user.js) though! :)
